lazyscribe/__init__.py

```diff
@@ -3,6 +3,7 @@
 from lazyscribe._meta import __version__  # noqa: F401
 from lazyscribe.experiment import Experiment
 from lazyscribe.project import Project
+from lazyscribe.repository import Repository
 from lazyscribe.test import Test
 
-__all__: list[str] = ["Experiment", "Project", "Test"]
+__all__: list[str] = ["Experiment", "Project", "Repository", "Test"]

```

Comment:
We should fix https://github.com/lazyscribe/lazyscribe/blob/018e211c63acc04f2eda54f533c447d81dad510b/tutorials/repository.py#L19 after this change.